cms/models_test.py

```diff
@@ -870,6 +877,34 @@ def test_external_course_page_properties():
     assert external_course_page.product == external_course_page.course
 
 
+def test_course_page_format_field_default_value():
+    """
+    Verifies that the "format" field in a course page has the default value FORMAT_ONLINE.
+    """
+    course_page = CoursePageFactory.create()
+
+    assert course_page.format == FORMAT_ONLINE
+
+
+@pytest.mark.parametrize("format_value", [FORMAT_ONLINE, FORMAT_OTHER])

```

Comment:
we don't usually use `_value` naming convention, You could simply use `format`.

Reply:
I added `_value` to the format to distinguish it from the built-in name because the test name also checks the values. If there are no problems with the shadow names, I can update them as needed.

Reply:
better to use `course_format` then.

Reply:
Sure.

Reply:
Addressed.